cbsodata/cbsodata3.py

```diff
@@ -217,7 +234,7 @@ def _select(select):
 
 
 def download_data(table_id, dir=None, typed=False, select=None, filters=None,
-                  catalog_url=None, proxies=None):
+                  catalog_url=None, **kwargs):
```

Comment:
This API will prevent the usage of kwargs for filters in the future without breaking backwards compatibility (such as [in django](https://docs.djangoproject.com/en/3.0/topics/db/queries/#chaining-filters)). Please consider that when making this design choice.

Reply:
Thanks for this feedback. I think I will implement something like this in the OData4 version #8. This OData3 API deprecates in 2020.

Reply:
I have a partial implementation of an OData query parser if anyone is interested (grammar + a number of abstract syntax tree manipulations to rewrite it).

Unfortunate it is part of a larger prototype that is not release ready - but ping me if there is interest.

Reply:
I'm welcoming all contributions to the OData version 4 API :)